nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -665,8 +665,13 @@ sub generateXserverConfig {
     mkpath($outDir, 0, 0755);
     write_file($fn, $hwConfig);
 
+    my $nixConfig = "";
     $fn = "$outDir/flake.nix";
     if ($flake) {
+        $nixConfig = <<EOF;
+  # Enable flakes to allow rebuilding the system.
+  nix.settings.experimental-features = [ "flakes" "nix-command" ];
```

Comment:
```suggestion
  nix.settings.experimental-features = [ "flakes" "nix-command" ];

```
lets avoid extra new lines in the final config